lua/starfall/libs_sh/entities.lua

```diff
@@ -829,6 +829,50 @@ function ents_methods:getBonePosition(bone)
 	return vwrap(pos), awrap(ang)
 end
 
+--- Returns the angle manipulate angle of the bone (local to the entity)
+-- @shared
+-- @param bone Bone index. (def 0)
+-- @return Manipulate angle of the bone
+function ents_methods:getManipulateBoneAngles(bone)
+	local ent = getent(self)
```

Comment:
Making the local variable `ent` is unnecessary as it's only used once in this and all other functions, same with `ang` here, although I guess the `ang` could be left for readability, but the whole line is short enough without it:
```lua
return awrap(getent(self):GetManipulateBoneAngles(bone))
```